src/Microsoft.DotNet.Build.VstsBuildsApi/VstsBuildHttpClient.cs

```diff
@@ -19,8 +20,17 @@ public VstsBuildHttpClient(JObject definition, VstsApiEndpointConfig config)
 
         protected override bool IsMatching(JObject localDefinition, JObject retrievedDefinition)
         {
+            string localDefinitionPath = localDefinition["path"].ToString();
```

Comment:
I prefer `.Value<string>()` rather than `.ToString()` because it's a more exact match, but I see `ToString` is used in this file already so I'm ok with it.